internal/pkg/manifest/applyenv_test.go

```diff
@@ -923,7 +923,7 @@ func TestApplyEnv_Image_HealthCheck(t *testing.T) {
 				}
 			},
 		},
-		"interval not overridden": {
+		"FIXED_AFTER_TRANSFORM_POINTER: interval not overridden": {
```

Comment:
What's the plan with these odd prefixes :D Is the idea to search for these strings afterwards and ensure that all of them are gone post pointer->struct refactor?

Reply:
Right! I also wanted to showcase whether the set of failing tests make sense to me and the reviewers, and whether the set of fixed tests makes sense. That way I can make sure I'm understanding the problem correctly, and my fix is indeed working in the way I expect them to be.